examples/tutorials/Quickstart Tutorial.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
I just saw that we actually have another Quickstart guide here: https://docs.aqueducthq.com/quickstart-guide, which is slightly different from (but very similar to) this notebook, which mirrors the example on the frontpage of our repo.

We should probably only have one of these quickstarts right? But I'm not sure how to merge them or which one to pick... The one on the website is written more like a guide and this one is written more like a quick example to get up and running Also, the one on the website talks about metrics, whereas this one doesn't. cc: @vsreekanti

Reply:
Sorry for the confusion. Anecdotally, the quickstart guide has been working pretty well for people who've jumped on calls with us, so feel free to update it with anything you feel is helpful from this, but I'd like to keep most of what's currently there if that's cool with you @kenxu95!

Reply:
ok sg - so I should translate the quickstart guide into a notebook then, and stick that in the "Example Workflows" section? as the "Quickstart Tutorial"

Reply:
yep!